manifests/init.pp

```diff
@@ -2,6 +2,10 @@
 #
 # @api public
 #
+# @param default_target
+#   The default systemd boot target, unmanaged if set to undef.
+#   Default: undef
```

Comment:
No need to write the default here, Puppet Strings puts that in REFERENCE.md automatically.